packages/swapper/src/swappers/zrx/getZrxTradeQuote/getZrxTradeQuote.ts

```diff
@@ -1,25 +1,22 @@
 import { SwapSource } from '@shapeshiftoss/types'
 import { AxiosResponse } from 'axios'
 
-import { GetTradeQuoteInput, TradeQuote } from '../../../api'
+import { GetTradeQuoteInput, SwapErrorTypes, TradeQuote } from '../../../api'
 import { getZrxMinMax } from '../getZrxMinMax/getZrxMinMax'
 import { ZrxPriceResponse } from '../types'
 import { bn, bnOrZero } from '../utils/bignumber'
 import { APPROVAL_GAS_LIMIT, DEFAULT_SOURCE } from '../utils/constants'
 import { normalizeAmount } from '../utils/helpers/helpers'
 import { zrxService } from '../utils/zrxService'
-import { ZrxError } from '../ZrxSwapper'
+import { ZrxSwapError } from '../ZrxSwapper'
 
 export async function getZrxTradeQuote(input: GetTradeQuoteInput): Promise<TradeQuote<'eip155:1'>> {
   const { sellAsset, buyAsset, sellAmount, sellAssetAccountId } = input
-  if (!buyAsset) {
-    throw new ZrxError('getQuote - Missing buyAsset')
-  }
-  if (!sellAsset) {
-    throw new ZrxError('getQuote - Missing sellAsset')
-  }
   if (buyAsset.chainId !== 'eip155:1' || sellAsset.chainId !== 'eip155:1') {
```

Comment:
use every on array of sellAsset and buyAsset vs  suggested`this.supportedChainIds` to determine this

Reply:
https://github.com/shapeshift/lib/issues/646